hphp/runtime/test/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+enable_testing()
```

Comment:
This should only be required if you use CMake's built-in testing support, but you aren't using it, so this isn't needed.